docs/getting_started/permissions.md

```diff
@@ -27,12 +27,22 @@ $ deno run --allow-read=/etc https://deno.land/std/examples/cat.ts /etc/passwd
 
 `--allow-write` works the same as `--allow-read`.
 
-This example restricts network access by whitelisting the allowed hosts:
+### Network access:
+
+_fetch.ts_:
 
 ```ts
 const result = await fetch("https://deno.land/");
 ```
 
+This is an example on how to whitelist hosts/urls:
+
+```shell
+$ deno run --allow-net=deno.land https://deno.land/std/examples/curl.ts https://deno.land/ fetch.ts
```

Comment:
@JavaScriptDude can you also include an example with comma separated list?
```
$ deno run --allow-net=deno.land,https://deno.land/std/examples/curl.ts,https://deno.land/ fetch.ts
```

Reply:
Sure thing. Just confirming on Discourse that its a stable feature.